src/components/common/TmSessionImportName.vue

```diff
@@ -7,7 +7,13 @@
       <div class="session-main bottom-indent">
         <Steps :steps="[`Recover`, `Name`, `Password`]" active-step="Name" />
         <TmFormGroup field-id="import-name" field-label="Your Address">
-          <p class="address">{{ importCosmosAddress }}</p>
+          <img
+            v-if="importedAddress !== {}"
```

Comment:
Shouldn't it be `importedAddress === {}`?

Reply:
haha lunie collective mind again, see my latest commit xD

Reply:
xD

Reply:
Did you push it?

Reply:
yep, I can see the changes here!

Reply:
Sorry, but I need to insist
```suggestion
            v-if="importedAddress === {}"
```